src/app/drive/services/download.service/fetchFileBlob.ts

```diff
@@ -26,6 +27,7 @@ export default async function fetchFileBlob(
     options: {
       notifyProgress: (totalBytes, downloadedBytes) => {
         options.updateProgressCallback(downloadedBytes / totalBytes);
+        options.updateNumItemsCallback && options.updateNumItemsCallback();
```

Comment:
Won't this run more than once per item download?

Reply:
I don't think so, this workflow comes specifically from firefox